client/signup/processing-screen/index.jsx

```diff
@@ -63,9 +64,14 @@ export class SignupProcessingScreen extends Component {
 	}
 
 	componentDidMount() {
+		const { flowName, localeSlug } = this.props;
+		const locale = localeSlug.split( /[-_]/ )[ 0 ];
+		if ( flowName !== 'onboarding' || ! includes( [ 'en', 'ja' ], locale ) ) {
```

Comment:
Small nitpick - why not use `[ 'en', 'ja' ].includes( locale )` instead of the lodash dependency?

Reply:
I didn't see a similar comment in the review above. Please ignore this :)